guide/private-key-management/single-user-schemes.md

```diff
@@ -0,0 +1,119 @@
+
+*{illustration}*
+
+**How it works** - A single private key is split into n key-shares that are distributed to several parties, devices or locations. When signing a transaction the required number of shares need to be coordinated into one valid signature.
+
+**Pros** - Can provide higher resistance to loss from theft and negligence
+
+**Cons** - Require precise coordination of key-shares when signing, few advantages over multi-key setups with Schnorr signatures, individual implementations not interoperable
+
+**When to use** - When target audience is knowledgeable and risk of theft is higher than negligance
+
+**When not to use** - When Schnorr signatures are available enabling multi-key setups
+
+**Variations** - Number of signatures required, location and distribution of pieces, signing procedure
+
+**Products that use this scheme** - ZenGo, Hexa, Bitfreeze
+
+### Multi-key
+All of the above schemes have relied on a single private key to control the wallet. This presents an all-or-nothing risk for loss of funds from both theft and negligence. To counter this a wallet can have several private keys attached of which all or a subset need to sign any transactions. This is often called multi-signature, or multisig for short and described as ‘n-of-n’ to indicate how many keys are needed to sign a transaction out of the issued number. For example a ‘2-of-3’ setup requires two of the three private keys to sign a transaction for it to be valid.
+
+This can raise the security since anyone needs access to more than one key to move any funds. But clearly it also increases complexity and the requirements on the user to keep even more keys securely stored and/or backed up.
+
+*{illustration}*
+
+**How it works** - A software wallet-product or coordination software initiates a multi-sig wallet that can have several private keys attached (co-signers), the user then adds private keys from other wallets generated elsewhere to the multisig after which the software wallet can complete the creation process. For any future transaction from the multi-sig wallet the required amount of co-signers need to sign (using Partially Signed Bitcoin Transactions - PSBT from [BIP174]) before any transaction is valid.
+
+**Pros** - Can add redundancy to private keys and increase security
+
+**Cons** - Adds complexity and op-sec burden for multiple private keys
+
+**When to use** - When storing large amounts, when users are likely to be very knowledgeable
```

Comment:
What counts as "very knowledgable"? I think this can be made more specific for example:

```suggesstion
**When to use** - When storing large amounts, when users are confident and able to manually safeguarding their private keys.
```